TypeCobol/Compiler/CupPreprocessor/ICompilerDirectiveBuilder.cs

```diff
@@ -138,9 +138,9 @@ public interface ICompilerDirectiveBuilder
         /// </summary>
         /// <param name="replaceTokn"></param>
         /// <param name="offToken"></param>
-        /// <param name="replacingOperands"></param>
+        /// <param name="cupReplacingOperands"></param>
         void EnterReplaceCompilerStatement(TypeCobol.Compiler.Scanner.Token replaceTokn,
-            TypeCobol.Compiler.Scanner.Token offToken, PairTokenListList replacingOperands);
+            TypeCobol.Compiler.Scanner.Token offToken, CupReplaceOperations cupReplacingOperands);
```

Comment:
LEADING and TRAILING are valid only on REPLACE operations not REPLACING ? I'm asking to understand the change, but it does not prevent us using `CupReplaceOperations` in copy replacing too.